webdriver/tests/find_element_from_shadow_root/find.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+
+from webdriver.transport import Response
+
+from tests.support.asserts import assert_error, assert_same_element, assert_success
+
+
+def find_element(session, shadow_id, using, value):
+    return session.transport.send(
+        "POST", "session/{session_id}/shadow/{shadow_id}/element".format(
+            session_id=session.session_id,
+            shadow_id=shadow_id),
+        {"using": using, "value": value})
+
+
+def test_found_element_equivalence(session, get_shadow_page):
```

Comment:
Mind moving this test down right before `test_find_element` so that we start with the null value test as we do for all the other tests?

Reply:
Done for find element and find elements in d74e42a.